src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,91 @@
+package parser;
+
+import commands.*;
+import exceptions.DukeException;
+
+public class Parser {
+    /**
+     * Returns a Command class based on input type.
+     * @param input An array of strings split by spaces.
+     * @return Command
+     */
+    public Command parseCommand(String[] input) throws DukeException {
+        StringBuilder name;
+        StringBuilder current;
+        String s1 = input[0];
+        switch (s1) {
```

Comment:
I'm not too sure if _switch_ statements can have line breaks between cases.  

https://se-education.org/guides/conventions/java/intermediate.html#naming